### DISCUSSION TITLE
Solver Crashes for Multidmensional Knapsack with equal Values

### DISCUSSION THREAD

**Discussion by vishhvak** [google/or-tools#3536] [Graph and Algorithms]
Hi, 

I'm trying to solve a nutritional diet problem where I have a list of items, each having calories, protein, carbs and fat as weights. I want to see which combinations of items' nutritional details add up to a diet requirement on calories, protein and carbs, and fat. 

I thought this would be a knapsack problem where I have equal values (no profit on each item), 4 dimensional weights and have 4 capacities as my limits. On trying a sample out and initializing the knapsack multidimensional bound and branch olver with equal values for all items makes the solver crash. Why is this the case?

Am i thinking of this problem incorrectly?

**Comment by Mizux** [comment]
Do you have a minimal reproducible example you can share with us (e.g. github gist ) ?

**Comment by lperron** [comment]
No news.
